atomic_reactor/plugin.py

```diff
@@ -134,26 +117,55 @@ def args_from_user_params(user_params: dict, /) -> dict:
         return {}
 
 
-class PluginsRunner(object):
+# Built-in plugins
+class PreBuildSleepPlugin(Plugin):
```

Comment:
can we remove this or rename this? maybe `TestPlugin` or `DebugPlugin` will be more clear?